Design.md

```diff
@@ -0,0 +1,96 @@
+4. A URL pointing to the location of OPERA frame image for that tile (see below)
+
+For example, the fields in metadata raster may look like:
+```
+OPERA_FRAMES: "1, 2"
+FRAME_1_REF_POINT: "129877.23, 128383.28"
+FRAME_1_EPSG: "12345"
+FRAME_1_REF_POINT_ARRAY: "10, 20"
+FRAME_1_REF_DATE: "2015/01/01"
+FRAME_1_SEC_DATE: "2015/01/02"
+FRAME_2_REF_POINT: "22348.23, 38973.28"
+FRAME_2_EPSG: "12345"
+FRAME_2_REF_POINT_ARRAY: "20, 11"
+FRAME_2_REF_DATE: "2015/02/01"
+FRAME_2_SEC_DATE: "2015/02/02"
+```
+
+In addition to this per-COG information, there will also be one accompanying metadata raster (in COG format) for each tile location (i.e., one per footprint). The metadata raster will have the same resolution, extent, and projection as the cumulative displacement COGs, and in fact will serve as the template for create cumulative displacement COGs. The metadata raster will have the following components
+1. A UInt16 band indicating the frame number the cumulative displacement value each pixel is sourced from
+2. A sidecar metadata file with the same information as the cumulative displacement sidecar files, except for secondary date and OPERA frame image URL
+
+**Note: a time-series stack of OPERA-DISP products from the same frame are not guaranteed to have the same reference date or location!**
+
+However having the same spatiotemporal reference for each pixel in a time-series stack is an important requirement for date/location re-referencing and velocity calcultions. Thus, we standardize the spatiotemporal reference for each product before mosaicking. 
+
+Note that we only standardize the spatiotemporal reference so that each location is consistent through time. We do not standardize this information across space. Non-continuous frame coverage, and varying data collection dates makes standardizing across space a difficult challenge that is planned to be undertaken during the creation of the OPERA-DISP Vertical Land Motion project.
+
+### 1x1 Degree Tile Creation
+Prior to mosaicking of OPERA-DISP products, we generate the metadata rasters.
+
+For each 1x1 degree tile, there is one metadata raster for each orbit direction, for a total of two per tile. The metadata rasters will each span a 1x1 degree area in the web mercator (EPSG:3857) projection with a 30 m pixel spacing. The frame band is created by cross-referencing each tile's extent with the OPERA-DISP frame footprints. A geospatial representation of the OPERA-DISP frame footprints can be created using the [OPERA burst_db utility](https://github.com/opera-adt/burst_db). Frames will be overlaid so that frames from the same relative orbit are contiguous. Relative orbits are stacked from East to West on the ascending pass (West on top), and from West to East on the descending pass (East on top). Within a relative orbit, frames are overlaid so that older frames are on top of younger frames. This corresponds to North on top for the ascending pass, and South on top for the descending pass.
```

Comment:
> Relative orbits are stacked from East to West on the ascending pass (West on top), and from West to East on the descending pass (East on top)

Using "biggest frame number on top" feels simpler to implement, and might give an equally pleasing mosaic. Along-track frames have consecutive frame ids.

Here's a rendering in qgis of what layering "biggest on top" would look like:
![Screenshot from 2024-09-23 09-43-31](https://github.com/user-attachments/assets/2a75027a-7ce9-413f-a941-8cef9496b57e)